packages/legend-application-query/src/stores/QueryBuilderPreviewDataHelper.ts

```diff
@@ -43,109 +28,82 @@ import { QueryBuilderAggregateOperator_Min } from './aggregateOperators/QueryBui
 import { QueryBuilderAggregateOperator_StdDev_Population } from './aggregateOperators/QueryBuilderAggregateOperator_StdDev_Population.js';
 import { QueryBuilderAggregateOperator_StdDev_Sample } from './aggregateOperators/QueryBuilderAggregateOperator_StdDev_Sample.js';
 import { QueryBuilderAggregateOperator_Sum } from './aggregateOperators/QueryBuilderAggregateOperator_Sum.js';
-import type { QueryBuilderAggregateOperator } from './QueryBuilderAggregationState.js';
-import { buildGetAllFunction } from './QueryBuilderLambdaBuilder.js';
-import { buildGenericLambdaFunctionInstanceValue } from './QueryBuilderValueSpecificationBuilderHelper.js';
+import { QueryBuilderSimpleProjectionColumnState } from './QueryBuilderProjectionState.js';
+import type { QueryBuilderState } from './QueryBuilderState.js';
+import {
+  COLUMN_SORT_TYPE,
+  SortColumnState,
+} from './QueryResultSetModifierState.js';
```

Comment:
Please don't forget to check the weird test failure caused by the rearrangement of imports

Reply:
Yes I did. this was due to circular dependencies due to us calling new Operators in the constructors of QueryBuilderState and AggregationState, which i fixed by moving the operators to a single loading file